elfpy/data/postgres.py

```diff
@@ -1,68 +1,37 @@
 """Initialize Postgres Server"""
 
+from __future__ import annotations
+
+import pandas as pd
 import sqlalchemy
-from sqlalchemy import Column, ForeignKey, Integer, String, create_engine
-from sqlalchemy.ext.declarative import declarative_base
-from sqlalchemy.orm import sessionmaker
+from sqlalchemy import create_engine
+from sqlalchemy.orm import Session, sessionmaker
 
-from elfpy.data.pool_info import PoolInfo
+from elfpy.data.db_schema import Base, PoolInfo, Transaction
 
 # classes for sqlalchemy that define table schemas have no methods.
 # pylint: disable=too-few-public-methods
 
-Base = declarative_base()
-
 # replace the user, password, and db_name with credentials
+# TODO remove engine as global
 engine = create_engine("postgresql://admin:password@localhost:5432/postgres_db")
 
 
-class UserTable(Base):
-    """User Schema"""
-
-    __tablename__ = "users"
-
-    # address
-    id = Column(String, primary_key=True)
-
-
-class TransactionTable(Base):
-    """Transactions Schema"""
-
-    __tablename__ = "transactions"
-
-    id = Column(String, primary_key=True)
-    user_id = Column(String, ForeignKey("users.id"))
-    amount = Column(Integer)
-
-
-class PoolInfoTable(Base):
-    """PoolInfo Schema"""
-
-    # names are in snake case to indicate values came from solidity
-    # pylint: disable=invalid-name
+# TODO figure out what this table is supposed to hold
```

Comment:
I guess this is a question for Jacob? But honestly if it's not being used elsewhere then I'd rather delete it instead of keeping it here in comments.